packages/core/src/components/checkbox/checkbox.scss

```diff
@@ -74,7 +74,13 @@
   }
 
   // indicating [disabled] input
-  .ods-checkbox:disabled + .ods-input-indicator::after {
-    border-color: helpers.color('content-disabled');
+  .ods-checkbox:disabled + .ods-input-indicator {
+    &::after {
+      border-color: helpers.color('content-disabled');
+    }
+
+    &::before {
+      background-color: helpers.color('background-disabled');
+    }
```

Comment:
I don't understand how this fixes it, seems to be the exact same rule/selector in a different file.
Is it just a specificity thing?

Reply:
It needs to come _after_ all other stylings like for "checked", as "disabled" should make the styling no matter what is applied previously.

It also _must_ be on each component styles individually, as indicator component is included first, after which follows all other additional CSS styles.

Reply:
I see, it is specificity then. Thanks.